src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -46,10 +46,10 @@ public void tearDown() {
     //Test where user has null timestamp
     public void getTimestampProperty_nullUserTimeStamp() {
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
-
+ 
```

Comment:
I think you added a bunch of spaces to this file for some reason. Removing them will make the PR much more readable and follow formatting guides anyway.

Reply:
Is there a general rule of thumb for spaces that adheres to the formatting guide lines?